geode-docs/tools_modules/http_session_mgmt/weblogic_setting_up_the_module.html.md.erb

```diff
@@ -91,7 +91,7 @@ If you are deploying an ear file:
     lib/fastutil-7.0.2.jar
     lib/geode-core-1.0.0.jar
     lib/geode-json-1.0.0.jar
-    lib/javax.transaction-api-1.2.jar
+    lib/javax.transaction-api-1.3.jar
```

Comment:
I don't really know what this file is documenting, but every other version here is wildly out of date

Reply:
These are the minimum changes to get modules working from a client. Yes other deps may be old but out of scope for this work.